L1Trigger/TrackFindingTracklet/src/L1SimTrack.cc

```diff
@@ -0,0 +1,37 @@
+#include "L1Trigger/TrackFindingTracklet/interface/L1SimTrack.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+using namespace std;
+using namespace trklet;
+
+L1SimTrack::L1SimTrack() {
+  eventid_ = -1;
+  trackid_ = -1;
+}
+
+L1SimTrack::L1SimTrack(
+    int eventid, int trackid, int type, double pt, double eta, double phi, double vx, double vy, double vz) {
+  eventid_ = eventid;
+  trackid_ = trackid;
+  type_ = type;
+  pt_ = pt;
+  eta_ = eta;
+  phi_ = phi;
+  vx_ = vx;
+  vy_ = vy;
+  vz_ = vz;
+}
+
+void L1SimTrack::write(ofstream& out) {
+  if (pt_ > -2.0) {
```

Comment:
What does negative pt_ mean?

Reply:
it means it is taking into account the charge.